src/Contracts/PermissionsTrait.php

```diff
@@ -19,7 +20,7 @@ trait PermissionsTrait
      *
      * @return bool
      */
-    public function assignRole(string $role) : bool
+    public function assignRole(string $role, Companies $company = null) : bool
```

Comment:
@rwhite27 this should be 

?Companies $company = null

Reply:
ok